src/main/java/uk/gov/hmcts/ccd/data/casedetails/JurisdictionMapper.java

```diff
@@ -13,7 +13,7 @@ public JurisdictionDisplayProperties toResponse(Jurisdiction jurisdiction) {
         result.setId(jurisdiction.getId());
         result.setName(jurisdiction.getName());
         result.setDescription(jurisdiction.getDescription());
-        result.setCaseTypes(jurisdiction.getCaseTypes());
+        result.setCaseTypeDefinitions(jurisdiction.getCaseTypeDefinitions());
```

Comment:
in this particular case adding the Definitions suffix will make things a bit confusing because we are creating a display object. Possible to revert this change?

Reply:
ideally we should not do this we should convert domain objects into DTO. Something to improve in the future